pkg/compactor/compactor.go

```diff
@@ -302,7 +330,15 @@ func newCompactor(
 	bucketClientFactory func(ctx context.Context) (objstore.Bucket, error),
 	blocksGrouperFactory BlocksGrouperFactory,
 	blocksCompactorFactory BlocksCompactorFactory,
+	limits Limits,
 ) (*Compactor, error) {
+	var remainingPlannedCompactions prometheus.Gauge
+	if compactorCfg.ShardingStrategy == "shuffle-sharding" {
```

Comment:
instead of using the hardcoded string, why not use ` util.ShardingStrategyShuffle`?